site/components/pages/Pledge/PledgeDashboard/Profile/index.tsx

```diff
@@ -90,6 +92,14 @@ export const Profile: FC<Props> = (props) => {
           concatAddress(props.pledge.ownerAddress)}
       </Text>
 
+      {!!profileUrl && (
+        <Text t="body1">
+          <a className="profileUrl" href={profileUrl}>
+            {prettifyUrl(profileUrl)}
+          </a>
```

Comment:
What do you think of opening the link in a new tab?
There's a silly wrapper component that does that 😆 
```
import { Anchor as A } from "@klimadao/lib/components";
```

Reply:
Sure, done 👍